web/tests/copy_and_paste.test.cjs

```diff
@@ -209,4 +217,70 @@ run_test("paste_handler_converter", () => {
+
+<span class="katex-display"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow></mrow><annotation encoding="application/x-tex"></annotation></semantics></math></span><span class="katex-html" aria-hidden="true"></span></span></span>
+
+<span class="katex-display"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow></mrow><annotation encoding="application/x-tex"></annotation></semantics></math></span><span class="katex-html" aria-hidden="true"></span></span></span>
+
+<span class="katex-display"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow></mrow><annotation encoding="application/x-tex"></annotation></semantics></math></span><span class="katex-html" aria-hidden="true"></span></span></span>
+
+<span class="katex-display"><span class="katex"><span class="katex-mathml"><math xmlns="http://www.w3.org/1998/Math/MathML" display="block"><semantics><mrow><mi>d</mi><mo>+</mo><mi>x</mi><mo>=</mo><mn>44</mn></mrow><annotation encoding="application/x-tex">d+x = 44
+</annotation></semantics></math></span><span class="katex-html" aria-hidden="true"><span class="base"><span class="strut" style="height:0.7778em;vertical-align:-0.0833em;"></span><span class="mord mathnormal">d</span><span class="mspace" style="margin-right:0.2222em;"></span><span class="mbin">+</span><span class="mspace" style="margin-right:0.2222em;"></span></span><span class="base"><span class="strut" style="height:0.4306em;"></span><span class="mord mathnormal">x</span><span class="mspace" style="margin-right:0.2778em;"></span><span class="mrel">=</span><span class="mspace" style="margin-right:0.2778em;"></span></span><span class="base"><span class="strut" style="height:0.6444em;"></span><span class="mord">44</span></span></span></span></span></p>
+<p>end</p>`;
+
+    assert.equal(
+        copy_and_paste.paste_handler_converter(input),
+        "intro text\n\n```math\na + b = 33\n\n\n\n\n\n\n\n\nd+x = 44\n``` \n\nend",
+    );
```

Comment:
I wonder if these KaTeX tests would be better written by just having each test case have the original source input, and passing that into KaTeX, rather than having a giant block of HTML in this file that's just the output of that. (We can just import it directly here). `web/tests/markdown.test.cjs`  has some logic you can look at it.

The result would be that you'd have something like:

```
const original_display_math_block = "a + b = 33\n\n\n\n\n\n\n\n\nd+x = 44\n"
input = "intro text\n\n" + katex.renderToString(original_katex_input) + "\n\nend";
...
```

Reply:
Is it just that you need to pass the `displayMode` parameter? `katex.renderToString(content, {displayMode: is_display})`